cloudformation/security-hq.template.yaml

```diff
@@ -386,6 +389,44 @@ Resources:
           Fn::FindInMap: [ Constants, App, Value ]
         PropagateAtLaunch: true
 
+  RemovePasswordExecutionFailureAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${SNSTopicForAlerts}
+      AlarmName: !Sub Security HQ failed to remove a vulnerable password
+      AlarmDescription: !Sub The credentials reaper feature of Security HQ logs either success or failure to cloudwatch, and this alarm lets us know when it logs a failure. Check the application logs for more details https://logs.gutools.co.uk/s/devx/goto/f9915a6e4e94a000732d67026cea91be
+      MetricName: IamRemovePasswordExecution
+      Namespace: SecurityHQ
+      Dimensions:
+        - Name: Status
```

Comment:
Could we call it something more specific than `Status`? It strikes me looking at your screenshots that the way cloudwatch drills down into metric dimensions makes `Status` a bit too generic because you end up with `SecurityHQ -> Status` which sounds like status of the whole app(?)